tests/userTest.js

```diff
@@ -903,7 +962,7 @@ describe('Main: Currently testing user management,', function () {
           expect(status).toBe(null)
         }).catch(error => {
           expect(typeof (error)).toBe('object')
-          expect(error.ocs.meta.statuscode).toEqual('102')
+          expect(error.ocs.meta.statuscode).toEqual('103')
```

Comment:
same as above

Reply:
I have tested manually, the status code is `103` when trying to remove a non-existing user from an existing group
Also, the documentation specifies it https://doc.owncloud.com/server/10.5/developer_manual/core/apis/provisioning-api.html#add-to-group